### ISSUE TITLE
Only works when the parent work item is open

### ISSUE THREAD

**Issue by ghost** [jpiquot/ChildTasksTemplate#9]
I really like this extension - it has been something my teams have asked for for years.

We're on Azure DevOps Server 2020.1

I can see Add tasks via the ... actions item in Backlogs, Boards, and Sprints, but if I try to use it in this scenario, it does not actually add the tasks. 

I have to open the work item, then choose Add tasks via the ... actions item.

Add tasks should not be available in the actions if it will not work in that context.

**Comment by o-o00o-o** [comment]
agree. If there isn't that level of control for an extension then perhaps an error could appear to tell you to go to the card view?

**Comment by rppwolfs** [comment]
Adding the option for it to work from  the ... actions item in Backlogs, Boards, and Sprint would be a great additional.
Could you please have a look
